manoseimas/mps_v2/models.py

```diff
@@ -96,7 +97,7 @@ def __unicode__(self):
 class GroupMembership(CrawledItem):
     member = models.ForeignKey(ParliamentMember)
     group = models.ForeignKey(Group)
-    position = models.CharField(max_length=64)
+    position = models.CharField(max_length=128)
     since = models.DateField(blank=True, null=True)
     until = models.DateField(blank=True, null=True)
 
```

Comment:
I found that keeping all models in one `core` app has many benefits:

- Usually it is good idea to separate *data logic* from *domain logic*, so if `manoseimas.core` would be responsible for *data logic* where all data models are defined and all other *domain logic* would go to separate apps depending on `core` it would be much easier to refactor models within single `core` app and *domain logic* between modelless apps. Moving modes between apps is not that trivial.

- In multi layered architecture, data layer usually is at the bottom and it means that everyone depends on models, so keeping them separate, will make it much easier to separate *domain logic*.

Of course, you can keep models together with app if that app has strong separation from whole project, like a subproject within a project.